### DISCUSSION TITLE
Compact Mode for VS Code like the one in IntelliJ IDE's

### DISCUSSION THREAD

**Discussion by Defcon404** [microsoft/vscode-discussions#612] [Extension Development QnA]
Is it possible to have something like a "Compact Mode" feature in VS code like it is now available in the latest version of IntelliJ products like IntelliJ and pycharm??
I think, the padding in between the sidebar/menu icons and the width of the sidebars/menu bars/status bar and the like should be minimized a bit so there is more room for the work area. I do know about the full-screen mode and vs code zen mode but they then hide partially or completely the menus etc unlike "compact mode" if this feature is made available in vs code as well.

Link to the pycharm compact mode feature:
https://www.jetbrains.com/help/pycharm/new-ui.html

**Comment by alefragnani** [comment]
Hi @Defcon404 ,

There is no such feature yet, but there is an open issue exploring UI density modes (https://github.com/microsoft/vscode/issues/168671), and a "Compact Mode" is present.

I suggest you to interact on that issue, adding your suggestions and references, so the VS Code team could take a look. 

Hope this helps

**Reply by Defcon404** [reply]
@alefragnani Thank you for the info. I will do so.

**Comment by gjsjohnmurray** [comment]
@Defcon404 please note that this discussion forum is intended for developers who are creating VS Code extensions. To report bugs or raise enhancement requests in VS Code please use https://github.com/microsoft/vscode/issues

**Reply by Defcon404** [reply]
@gjsjohnmurray Thank you for the info. I will do so.
